api/data/network-coinlookups.js

```diff
@@ -9,11 +9,6 @@ const terraCoinLookup = [
     viewDenom: 'KRT',
     chainToViewConversionFactor: 1e-6
   },
-  {
```

Comment:
why did this one got deleted?

Reply:
It was a duplicate and destroying the index as a valid key-value reference